etna/collections/models.py

```diff
@@ -288,9 +314,23 @@ class TimePeriodExplorerPage(AlertMixin, BasePageWithIntro):
     body = StreamField(
         TimePeriodExplorerPageStreamBlock, blank=True, use_json_field=True
     )
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
     start_year = models.IntegerField(blank=False)
     end_year = models.IntegerField(blank=False)
     content_panels = BasePageWithIntro.content_panels + [
+        MultiFieldPanel(
```

Comment:
Why not just use a single `FieldPanel` here?

Reply:
Ah, sorry Andy! I meant to put some context in this PR - we're going to add a "hero image caption" at some point once it's been spec'd out. I thought I'd just put this here to get it started, if that's okay? I did it for the last hero image addition too